RocketChat/Rocket.Chat
# [FIX] Leave team modal crashing, remove member modal i18n

## Proposed changes (including videos or screenshots)


Update some labels and styles to better comply with the design. Fixed a crashing modal when trying to leave a team without the `view-all-team-channels` permission.


## Issue(s)


## Steps to test or reproduce

- As a user without admin or owner/leader permission, enter a team.
- Try to leave the team
- You should be able to leave

## Further comments
